### PR TITLE
Add tags to people page.

### PR DESCRIPTION
**Pull request by lostcarpark**
Have added tags for people as follows:
- Added tags to people file format (tags take same format as program tags).
- Added TAG options under PEOPLE in config. Takes same format as program tag settings, except doesn't have DAY_TAG options because that would make no sense.
- Added TagSelector component to People page, and filter people by tags.
- Added tag listing under name on Person page.
- Added CSS settings for formatting person tags.

Not related to tags, but have added back button to person page, as there wasn't a clear route back previously, and pressing browser back button was a little clunky.

### CODE REVIEW

**Comment by LVerhulst4321** [comment]
Awesome! I look forward to checking it out.

File: README.md [hunk 1]
```diff
@@ -105,6 +105,11 @@ The main place customisations go is the `src/config.json` file. Settings current
 - `PEOPLE.THUMBNAILS.DEFAULT_IMAGE`: Set to default thumbnail for participants with no photo. Can be filename of image in public directory, or external URL. Leave blank for no default thumbnail.
 - `PEOPLE.SORT.SHOW_CHECKBOX`: Set to false to hide "Sort by full name" checkbox. Useful if your data only contains "name for publications".
 - `PEOPLE.SORT.CHECKBOX_LABEL`: Label for "Sort by full name" checkbox.
+- `PEOPLE.TAGS.PLACEHOLDER`: The placeholder when selecting person tags (unless separated).
+- `PEOPLE.TAGS.SEARCHABLE`: Whether the tag list can be searched by typing (unless separated).
+- `PEOPLE.TAGS.HIDE`: If true, hide the tags drop-down. Tags still displayed on items.
+- `PEOPLE.TAGS.SEPARATE`: An array of tag prefixes to separate into individual drop-downs, and if drop-down is searchable or hidden. Tags should be specified as follows: `{ "PREFIX": "type", "PLACEHOLDER": "Select type", "SEARCHABLE": true|false, "HIDE": true|false }`.
+- `PEOPLE.TAGS.FORMAT_AS_TAG`: If set to true, turns Grenadine item format into a KonOpas-style "type" tag.
```
Comment (LVerhulst4321) [inline README.md:112]: FORMAT_AS_TAG is only used on program items, so it's not needed here.

File: src/config.json [hunk 2]
```diff
@@ -115,6 +115,14 @@
       "SHOW_CHECKBOX": true,
       "CHECKBOX_LABEL": "Sort by full name"
     },
+    "TAGS": {
+      "PLACEHOLDER": "Select tags",
+      "SEARCHABLE": false,
+      "HIDE": false,
+      "SEPARATE": [
+      ],
+      "FORMAT_AS_TAG": false
```
Comment (LVerhulst4321) [inline src/config.json:124]: FORMAT_AS_TAG is only used on program items, so it's not needed here.

**Comment by LVerhulst4321** [comment]
I made a couple of notes in the code review. FORMAT_AS_TAG is only used on program items with the optional format field.

**Comment by lostcarpark** [comment]
Thanks. I have removed FORMAT_AS_TAG from the config file and from the ReadMe. However, the same code runs for both, so in theory someone could use it if they wanted to.

**Review by LVerhulst4321 — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (lostcarpark) [reply README.md]: Removed FORMAT_AS_TAG from People in ReadMe.

File: src/config.json (hunk 2 above)
Comment (lostcarpark) [reply src/config.json]: Removed FORMAT_AS_TAG from People in config.

**Comment by mcdemarco** [comment]
The config file now points to local program and people data files that don't appear to be in the repository.

**Pull request merged**